tutorials/tts/Vits_Training.ipynb

```diff
@@ -0,0 +1,430 @@
+    "  - Sampling rate of 22050Hz or higher\n",
+    "  - Speech should contain a variety of speech phonemes\n",
+    "  - Audio split into segments of 1-10 seconds\n",
+    "  - Audio segments should not have silence at the beginning and end\n",
+    "  - Audio segments should not contain long silences inside\n",
+    "\n",
+    "After obtaining the speech data and splitting into training, validation, and test sections, it is required to construct .json files to tell NeMo where to find these audio files.\n",
+    "\n",
+    "The .json files should adhere to the format required by the `nemo.collections.tts.data.dataset.TTSDataset` class. For example, here is a sample .json file\n",
+    "\n",
+    "```json\n",
+    "{\"audio_filepath\": \"/path/to/audio1.wav\", \"text\": \"the transcription\", \"duration\": 0.82}\n",
+    "{\"audio_filepath\": \"/path/to/audio2.wav\", \"text\": \"the other transcription\", \"duration\": 2.1}\n",
+    "...\n",
+    "```\n",
+    "Please note that the duration is in seconds.\n",
+    "\n",
+    "\n",
+    "Then you are ready to run your training script:\n",
+    "```bash\n",
+    "python vits.py train_dataset=YOUR_TRAIN.json validation_datasets=YOUR_VAL.json trainer.devices=-1\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluating VITS\n",
+    "\n",
+    "Let's evaluate the quality of the VITS model.\n",
+    "\n",
+    "VITS is end-2-end model, so we don't need any additional models to generate audios."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from matplotlib.pyplot import imshow\n",
+    "from matplotlib import pyplot as plt\n",
+    "import IPython.display as ipd\n",
+    "import numpy as np\n",
+    "import torch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seed = 1337\n",
+    "\n",
+    "def evaluate_vits_ckpt(model, test):\n",
+    "    with torch.no_grad():\n",
+    "        torch.manual_seed(seed)\n",
+    "        torch.cuda.manual_seed(seed)\n",
+    "        torch.backends.cudnn.enabled = True\n",
+    "        torch.backends.cudnn.benchmark = False\n",
+    "        tokens = model.parse(str_input = test)\n",
+    "        audio = model.convert_text_to_waveform(tokens = tokens)\n",
+    "\n",
+    "\n",
+    "    audio = audio.to('cpu').numpy()\n",
+    "    audio = audio / np.abs(audio).max()\n",
+    "    return audio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test = \"Hey, this produces speech!\"\n",
+    "\n",
+    "model = VitsModel.from_pretrained(\"tts_en_lj_vits\")\n",
+    "\n",
+    "audio = evaluate_vits_ckpt(model, test)\n",
+    "\n",
+    "print(\"predicted audio\")\n",
+    "ipd.display(ipd.Audio(audio, rate=22050))"
```

Comment:
Adding a section for model evaluation is excellent, but only displaying synthesized audio signals is not enough to understand how far the difference from original audio. Could you please also help adding a mel-spectrogram comparison between synthesis and original?